lib/servers/stepper_motor_server/stepper_motor_server.py

```diff
@@ -0,0 +1,132 @@
+# Created on Feb 12, 2017
+# @author: Anthony Ransford
+
+"""
+### BEGIN NODE INFO
+[info]
+name = stepper_motor_server
+version = 1.0
+description = drives stepper motors
+instancename = stepper_motor_server
```

Comment:
probably good to be more specific in the server name.  I can easily see adding more stepper motor servers.